ext/tm/include/tm/string.hpp

```diff
@@ -1452,6 +1452,29 @@ class String final {
         m_str[m_length] = '\0';
     }
 
+    /**
+     * Grow the capacity (allocated memory) of the string.
+     *
+     * ```
+     * auto str = String { "abc" };
+     * str.set_capacity(100);
+     * assert_eq(100, str.capacity());
+     * assert_eq(3, str.size());
+     * ```
+     *
+     * It does not change the capacity if the new capacity is smaller than the current size
+     *
+     * ```
+     * auto str = String { "abc" };
+     * str.set_capacity(1);
+     * assert_eq(3, str.capacity());
+     * assert_eq(3, str.size());
+     * ```
+     */
+    void set_capacity(size_t new_size) {
+        grow_at_least(new_size);
```

Comment:
What if we rename `grow_at_least` to `set_capacity`? I don't think we need to maintain any backward compatibility since no one else is using the library. 😉

Reply:
Sorry I didn't see the commit message. Sounds good for now. Carry on!